h2o-py/h2o/estimators/stackedensemble.py

```diff
@@ -155,10 +159,27 @@ def __init__(self,
                random number)
                Defaults to ``-1``.
         :type seed: int
+        :param score_each_iteration: Whether to score during each iteration of model training.
```

Comment:
What about score_tree_interval for tree-based models?

Reply:
Thanks, I forgot about that one.

It makes me wonder if this PR is a good idea. We can already set the parameters via `metalearner_params` and exposing parameters that are applicable only on some meta learners on the SE level doesn't seem like a good idea to me.

Reply:
@tomasfryda I thought the ticket was only about `stopping_*` params that define the early stopping feature and that are supported by all metalearners I think.
I'm much less convinced about `score_each_iteration` or `score_tree_interval` that are not really necessary to use early stopping, just optional additions.

Reply:
Ok, I thought it should include even these as I think they influence the stopping - to be able to early stop we have to have scored iterations/trees, in other words, it defines the "resolution" used by the early stopping. At least this is how it worked in GLM last time I looked.

I will remove the `score_each_iteration`.

Reply:
@sebhrusen Without those parameters, the early stopping feature is not reproducible and I would consider it problematic in general. If you don't know how often H2O scores then it is very difficult to correctly set stopping rounds.

Reply:
@michalkurka I get your point about reproducibility. My concern is purely semantic, and deals with API choices.

To me the problem is similar to the decision that was made long ago to expose only the `stopping_*` params in 
AutoML: I guess that only those ones have been exposed to AutoML because they are enough to define the early stopping feature. The others depend on each model trained internally. If the user wanted to specify the optional ones for each algo, now s·he could override those using `algo_parameters`. Not ideal, but otherwise we could as well expose any possible algo param as a first-class param.

For SE, I think it's similar: the `stopping_*` are enough to define the feature—a feature otherwise made available to all algos, except SE until now—, but if the user needs to fine tune, s·he will have to consider the specific metalearner used by the SE, and may as well set those params as `metalearner_params`.

Reply:
@ledell wdyt?

Reply:
I think the difference between AutoML and SE is that AutoML by default has predictable scoring intervals and users can reason about it and use early stopping to their advantage.

I am thinking that these parameters are not needed on SE level because without understanding what metalearner is used and what parameters goes in the ability to set the early stopping parameters is limited anyway.

Reply:
@sebhrusen @tomasfryda Let's chat on Slack to discuss in more detail and then return to the thread here...

Reply:
@michalkurka we agreed on not to expose the params at all, and as a result @tomasfryda will close this PR.
It was based on a misconception that some of the high-level params were already forwarded to the metalearner, but this is actually not the case: only data/input-related params are, not algo-related ones.

Thanks for being watchful on this.